### ISSUE TITLE
Missing Export In Browser Index.js

### ISSUE THREAD

**Issue by wbotelhos** [fnando/i18n#92]
## Description

Missing export in `browser/index.js`

## How to reproduce

```js
import { I18n } from 'i18n-js';
```

## What do you expect

Import the `I18n`.

## What happened instead

It's returning void.

## Software:

- Gem version: 4.2.3
- Ruby version: 3.2.3

## Full backtrace

```text
18:54:16 js.1       | ▲ [WARNING] Import "I18n" will always be undefined because the file "../../node_modules/i18n-js/dist/browser/index.js" has no exports [import-is-undefined]
18:54:16 js.1       | 
18:54:16 js.1       |     modules/i18n.js:1:9:
18:54:16 js.1       |       1 │ import { I18n } from 'i18n-js';
```

**Comment by fnando** [comment]
what bundler tool are you currently using?

**Comment by wbotelhos** [comment]
I'm using `esbuild` `0.20.0`.

Here is my config:

```
import { I18n } from 'i18n-js';

const locale = document.documentElement.lang;

import en from '@/translations/en.json';
import ptBR from '@/translations/pt-BR.json';
import ru from '@/translations/ru.json';

const i18n = new I18n();

i18n.store(
  {
    'en': en,
    'pt-BR': ptBR,
    'ru': ru,
  }[locale]
);

i18n.enableFallback = true;
i18n.defaultLocale = document.documentElement.dataset.defaultLocale;
i18n.locale = locale;

export default i18n;
```

**Comment by fnando** [comment]
oh, this has been covered on the readme. Go to the troubleshooting section. :)

**Comment by wbotelhos** [comment]
It fixed and another lib `@glidejs/glide` didn't like it. Maybe I need understand better how all those configs works before upgrade. Any tip here?

```
19:12:07 js.1       | ✘ [ERROR] No matching export in "../../node_modules/@glidejs/glide/dist/glide.js" for import "default"
19:12:07 js.1       | 
19:12:07 js.1       |     pages/events/views/events/tickets.js:3:7:
19:12:07 js.1       |       3 │ import Glide from '@glidejs/glide';
19:12:07 js.1       |         ╵        ~~~~~
```

**Comment by fnando** [comment]
I'll take a look at this tonight.

**Comment by fnando** [comment]
Ah, looks like we can't assume all packages will export `main` as a module export. Try moving `main` further down the list on `--main-fields`, like `--main-fields=module,main,browser`

**Comment by fnando** [comment]
Yep, was able to reproduce this locally. I updated the readme with this new info.

**Comment by wbotelhos** [comment]
It worked, Nando. Thanks! ❤️

**Comment by wbotelhos** [comment]
@fnando Unfortunately I got another error, this time it comes from [Tippy.js](https://github.com/atomiks/tippyjs). It's not recognizing the var `process`.

```
ReferenceError: process is not defined
    at event.debug-b5100f8117412a9eab692445c8a1940819c880ee47994a770182f1836e8a0846.js:56946:1
(anonymous) @ event.debug-b5100f8117412a9eab692445c8a1940819c880ee47994a770182f1836e8a0846.js:56946
```

It happens during the import:

```js
import tippy from 'tippy.js';
```

It's on this part of the code:

```js
var visitedMessages;
if (process.env.NODE_ENV !== "production") { <<<
  resetVisitedMessages();
}
```

**Comment by fnando** [comment]
Oh, i got this one before. I think you can use something like this: `--define:process.env.NODE_ENV="development"` (or production)

**Comment by wbotelhos** [comment]
It worked, Nando. Again, thank you!

**Comment by wbotelhos** [comment]
Hey, @fnando

I'm no sure if I should keep bringing it to this issue, since it can be outside this lib scope. Let me know if it does not make sense from now on.

I found another problem with [tippyjs](https://github.com/atomiks/tippyjs):

```
RangeError: Maximum call stack size exceeded
    at getWindow (system.debug-faf277173fc11b1de6a001ffccc0787d888b747577cb92cdc1b2d61e49d22bb3.js:26806:19)
    at getComputedStyle (system.debug-faf277173fc11b1de6a001ffccc0787d888b747577cb92cdc1b2d61e49d22bb3.js:26999:10)
```

Looks like there is an [issue](https://github.com/Leaflet/Leaflet/issues/7127) related to it. For now I'll rollback and wait for a better time to upgrade just it.

**Comment by fnando** [comment]
I'm thinking about reverting the package.json change because it broke too many projects, unfortunately. 🫠

**Comment by fnando** [comment]
Just released v4.4.3-beta.0. Can you please give it a try and report back? Thanks!

**Comment by wbotelhos** [comment]
Now It's working correctly. Hope you can find a full supported solution in the future. Thanks!

**Comment by fnando** [comment]
just released v4.4.3
